tests/unit_tests/db/test_db_handler.py

```diff
@@ -76,360 +163,239 @@ def test_find_latest_simulation_model_db(db, db_no_config_file, mocker):
         db_copy._find_latest_simulation_model_db()
 
     db_names = [
-        "CTAO-Simulation-Model-v0-3-0",
-        "CTAO-Simulation-Model-v0-2-0",
-        "CTAO-Simulation-Model-v0-1-19",
-        "CTAO-Simulation-Model-v0-3-9",
-        "CTAO-Simulation-Model-v0-3-19",
-        "CTAO-Simulation-Model-v0-3-0",
-        "CTAO-Simulation-Model-v0-3-0-alpha-2",
-        "CTAO-Simulation-Model-v0-4-19-alpha-1",
-        "CTAO-Simulation-Model-v0-4-19-dev1",
+        "CTAO-Simulation-ModelParameters-v0-3-0",
+        "CTAO-Simulation-ModelParameters-v0-2-0",
+        "CTAO-Simulation-ModelParameters-v0-1-19",
+        "CTAO-Simulation-ModelParameters-v0-3-9",
+        "CTAO-Simulation-ModelParameters-v0-3-19",
+        "CTAO-Simulation-ModelParameters-v0-3-0",
+        "CTAO-Simulation-ModelParameters-v0-3-0-alpha-2",
+        "CTAO-Simulation-ModelParameters-v0-4-19-alpha-1",
+        "CTAO-Simulation-ModelParameters-v0-4-19-dev1",
     ]
     mocker.patch.object(db_copy.db_client, "list_database_names", return_value=db_names)
-    db_copy.mongo_db_config["db_simulation_model"] = "CTAO-Simulation-Model-LATEST"
+    db_copy.mongo_db_config["db_simulation_model"] = "CTAO-Simulation-ModelParameters-LATEST"
     db_copy._find_latest_simulation_model_db()
-    assert db_copy.mongo_db_config["db_simulation_model"] == "CTAO-Simulation-Model-v0-3-19"
+    assert (
+        db_copy.mongo_db_config["db_simulation_model"] == "CTAO-Simulation-ModelParameters-v0-3-19"
+    )
+
 
+def test_get_model_parameter(db, mocker):
+    """Test get_model_parameter method."""
+    mock_read_mongo_db = mocker.patch.object(
+        db, "_read_mongo_db", return_value={"parameter": "value"}
```

Comment:
Does it really return a dictionary with this structure of just `{"parameter": "value"}`? If not, I suggest to make this a bit more realistic. It doesn't matter for the test since it is just mocking it, but it helps in understanding the code by looking at the test.
(There are more cases below.)

Edit: After seeing the amount of cases with random values below, I won't insist. This one feels different cause it's about structure rather than values, but OK.

Reply:
But this is more than sufficient for the tests we are running here. The full dict is really not necessary. Let me know if you disagree.

Reply:
As I wrote in the edit, it would become too cumbersome to change everywhere as there are many cases. The only reason I brought it up is because sometimes I read the test in order to understand the code/input/output. In this case it doesn't help to do it. It is not super important though I figured it out in the end, so you can leave it as is.